### ISSUE TITLE
onConfirm receives all zero values

### ISSUE THREAD

**Issue by hugochinchilla** [troberts-28/react-native-timer-picker#26]
No matter what value I select on the modal, the onConfirm callback always receives all zeros.

This is how I'm using it:

```
<TimerPickerModal
    visible={showCancelTimePicker}
    initialValue={{
        hours: cancelTime.getHours(),
        minutes: cancelTime.getMinutes(),
        seconds: 0,
    }}
    setIsVisible={setShowCancelTimePicker}
    onConfirm={(pickedDate) => {
        console.warn("SAVING!");
        console.log(pickedDate, "pick");
        setShowCancelTimePicker(false);
    }}
    modalTitle="Margen de cancelación"
    confirmButtonText={"Guardar"}
    hideCancelButton={true}
    onCancel={() => setShowCancelTimePicker(false)}
    closeOnOverlayPress
    hideSeconds={true}
    styles={{
        theme: "light",
    }}
/>
```

**Comment by troberts-28** [comment]
Thank you for reporting this @hugochinchilla. It was a knock-on of a big refactor I did in the previous version.

Now fixed in `v1.8.1`!

**Comment by hugochinchilla** [comment]
Thank you, I will try it this weekend and report if I find any problems.
